shared/macros.jinja

```diff
@@ -679,3 +740,23 @@ openssl()
 )
 
 {{%- endmacro %}}
+
+{{% macro machineconfig_description_footer() -%}}
+<p>
+Note that this needs to be done for each <tt>MachineConfigPool</tt>
+</p>
+<p>
+For more information on how to configure nodes with the Machine Config
+Operator see
+{{{ weblink(link="https://docs.openshift.com/container-platform/4.6/post_installation_configuration/machine-configuration-tasks.html",
+            text="the relevant documentation") }}}.
+</p>
+{{% endmacro %}}
+
+{{% macro rhcos_node_login_instructions() -%}}
+    As a user with administrator privileges, log into a node in the relevant pool:
+    <pre>
+    $ oc debug node/$NODE_NAME
+    sh-4.4# chroot /host
```

Comment:
This step might be clearer if we say:

At the `sh-4.4#` prompt, run:
# chroot /host

Reply:
could you propose a snippet? I'm not sure what you mean :confused: